taskt/Core/Server/WebSocketClient.cs

```diff
@@ -17,63 +20,61 @@ namespace taskt.Core.Server
     /// </summary>
    public static class SocketClient
     {
-        public static UI.Forms.frmScriptBuilder associatedBuilder;
-        private static string publicKey;
-        private static string serverURI;
-        private static DateTime connectionOpened;
-        private static WebSocket4Net.WebSocket webSocket;
-        public static string connectionException;
-        private static bool retryOnFail;
-        private static System.Timers.Timer reconnectTimer;
-        private static bool bypassCertificationValidation;
-        public static Serilog.Core.Logger socketLogger;
+        public static frmScriptBuilder AssociatedBuilder;
+        public static string ConnectionException;
+        public static Logger SocketLogger;
+
+        private static string _publicKey;
+        private static string _serverURI;
+        private static DateTime _connectionOpened;
+        private static WebSocket _webSocket;
+        private static bool _retryOnFail;
+        private static System.Timers.Timer _reconnectTimer;
```

Comment:
Please remove namespace

Reply:
There's a conflict between `Windows.Forms.Timer` and `System.Timers.Timer` so I decided to just leave them. The other option would be to alias the namespaces which could be worth it if there was some meaningful name apart from `SystemTimer` or `FormsTimer`, but I think having the full assembly carries the meaning just as well. What do you think?

Reply:
I vote to leave it as it is if there's an overlap. These things happen often enough.